fixtures/ssr/src/components/FormElementTests/InputElements.js

```diff
@@ -0,0 +1,24 @@
+// as specified https://github.com/facebook/react/issues/9866
+import React, {Component} from 'react';
+
+export default class InputElements extends Component {
+  state = {text: 'initialText'};
+  handleChange = e => {
+    this.setState({text: e.target.value});
+  };
+  render() {
+    return (
+      <div>
+        <h3> {"<input>"} Elements </h3>
+        <p>
+          Uncontrolled: <input placeholder="Uncontrolled input" />
+        </p>
+        <p>
+          Controlled: <input value={this.state.text} onChange={this.handleChange} />
+          <br />
+          text: {this.state.text}
+        </p>
+      </div>
+    );
+  }
+}
```

Comment:
For the DOM test fixtures, we use the following format:

<img width="934" alt="screen shot 2017-11-13 at 9 18 25 pm" src="https://user-images.githubusercontent.com/590904/32759610-3f5e6420-c8b8-11e7-99c7-1aeb80978d83.png">

What do you think about including instructions and expected behavior as a part of these fixtures? Is that appropriate for these types of tests?

Reply:
ooh, this is pretty. one concern is that this may be too verbose because we have a -lot- to cover here. but i have no qualms incorporating this first and then deleting after if we dont like it.